### DISCUSSION TITLE
add scan path no longer saved

### DISCUSSION THREAD

**Discussion by stableartai** [zanllp/infinite-image-browsing#706] [Q&A]
Since update it appears the scan path is not being saved for area other than root output folder.

**Comment by zanllp** [comment]
Please paste the displayed error message, which might appear in the terminal when starting Python or in the browser's console.

**Comment by zanllp** [comment]
I just tested it and there is no problem. Please ensure that the path you added previously still exists. 

One thing to note: if you move the folder and IIB finds that the saved scan path is no longer accessible, it will clean up the related records. Please first check if this is the case.

**Comment by stableartai** [comment]
So looks like the path is there and the new one I created because before didn't see any thumbnails previews. Will monitor as any index should have now been well completed and since no new items will be in that path, shouldn't have to worry about a resync. Will reboot system soon, but normally workstation can stay up for weeks unlike a windows OS.
